tests/settings.py

```diff
@@ -10,3 +10,7 @@
 # Reconfigure pulse to operate on default vhost of rabbitmq
 PULSE_URI = BROKER_URL
 PULSE_EXCHANGE_NAMESPACE = 'test'
+
+# Set a fake api key for testing bug filing
+BZ_API_KEY = "12345helloworld"
```

Comment:
This doesn't work yet because I can't seem to register the responses stuff in test_bugzilla.py below. If you set this to an actual api key from bugzilla-dev, running the test will pass and create a new bug.